src/tools/create-app-tester.js

```diff
@@ -46,14 +46,15 @@ const createAppTester = appRaw => {
     bundle = bundle || {};
 
     const method = resolveMethodPath(appRaw, methodOrFunc);
+    let myStoreKey = storeKey ? `testKey-${storeKey}` : `testKey-${method}-${randomSeed}`
```

Comment:
though it's perfect for quick adhoc use, i'd probably name this variable something else. maybe `customStoreKey`.

That said, looking below, I'd probably move the ternary up here and rename that variable. So the whole thing would be

```js
(appRaw, customStoreKey) => 
// ...
const storeKey = shouldPaginate(appRaw, method)
  ? storeKey
    ? `testKey-${storeKey}`
    : `testKey-${method}-${randomSeed}`
  : undefined;

// ...
  method,
  bundle,
  storeKey
};
```

if you go that route, remember to adjust the `index.d.ts` appropriately.

Reply:
good idea, I like your suggested variable name better.